### ISSUE TITLE
Default Theme Disappear after switching to other theme

### ISSUE THREAD

**Issue by aakash83** [moxi9/phpfox#529]
Recently I am across this situation When I changed theme from default to legacy and set it to default. After this when i want to  change my theme back to default I can't see it in admincp>> themes. I am sure I did not deleted it . Again  when i  tested it on my dev site the issue remains the same.  If we install any other theme(when our current theme is other than defualt) and set it to default the previous theme from which you are changing  still remains in admincp. Now there is no way to switch back to defualt theme.

**Comment by Data66PF** [comment]
Please try upgrading to v4.0.8 and see if the issue resolves as I don't have any issues switching themes.

**Comment by aakash83** [comment]
upgraded to 4.08 but  still having same issue. I can install new themes and all the installed themes are appearing in admincp but not the default one.

**Comment by Data66PF** [comment]
http://scripttechs.com/article/363/reinstall-v4-default-theme/ that tutorial could help you get it back if maybe you deleted it but I'll test more on my development site if you say it deleted itself.

**Comment by aakash83** [comment]
it was not deleted because i have this problem on my live site and on localhost as well.

**Comment by Data66PF** [comment]
I can't reproduce it. I did this:

1. Set Legacy as default theme.
2. Get another theme (I chose Alizaren) and set it as default
3. Look in my list of themes (I have 6 of them now) and Alizaren is default at the top. At the bottom of the list is Neutron which is the default theme that comes with the PHPfox script. 
4. Now I set Neutron back to default. 

This appears to work. Is there some other step?
